master/src/ComputerMonitoringWidget.h

```diff
@@ -52,6 +53,8 @@ class ComputerMonitoringWidget : public FlexibleListView, public ComputerMonitor
 		m_ignoreWheelEvent = enabled;
 	}
 
+	QTimer t_mousePressAndHold;
```

Comment:
Please change the prefix to `m_` (member).

Reply:
Okay. Done.